{{ cookiecutter.project_name }}/{{ cookiecutter.project_name }}/settings_test.py

```diff
@@ -13,6 +13,12 @@
 import semantic_version
 
 
+REQUIRED_SETTINGS = {
+    'MAILGUN_URL': 'http://fake.mailgun.url',
+    'MAILGUN_KEY': 'fake_mailgun_key',
+}
+
+
 class TestSettings(TestCase):
```

Comment:
Could you backport your test in MM for `MAILGUN_*` settings to here too?

Reply:
Yes